app/src/main/java/dev/davidv/translator/LanguageDetector.kt

```diff
@@ -40,4 +44,33 @@ class LanguageDetector {
         null
       }
     }
+
+  suspend fun detectLanguageRobust(
```

Comment:
In which situations does this help? If the language is not detected initially, does giving it some other hints nudge it to a valid answer?

Reply:
> I found that looping through downloaded models kind of make the success rate higher

Interesting, I didn't expect that. Sounds like a potential bug in CLD

> Is there a better way to achieve the same thing?

not that I know of